src/main/java/forestry/storage/ModuleCrates.java

```diff
@@ -37,6 +45,11 @@
 @ForestryModule(moduleID = ForestryModuleUids.CRATE, containerID = Constants.MOD_ID, name = "Crate", author = "SirSengir", url = Constants.URL, unlocalizedDescription = "for.module.crates.description")
 public class ModuleCrates extends BlankForestryModule {
 
+	private static final String CONFIG_CATEGORY = "crates";
+
+	public static final List<String> cratesRejectedOreDict = new ArrayList<>();
+	public static List<ItemStack> cratesRejectedItem = new ArrayList<>();
```

Comment:
Can you please change this to a map with an `Item` key and and a `List` value or you use a `Multimap`. This way you not have to iterate over every stack even it the stack has nothing todo with the item.